build/yamls/elk-flow-collector/logstash/ipfix.yml

```diff
@@ -183,3 +183,27 @@
   144:
     - :string
     - :destinationPodLabels
+  145:
+    - :uint64
+    - :throughput
+  146:
+    - :uint64
+    - :reverseThroughput
+  147:
+    - :uint64
+    - :throughputFromSourceNode
+  148:
+    - :uint64
+    - :throughputFromDestinationNode
+  149:
+    - :uint64
+    - :reverseThroughputFromSourceNode
+  150:
+    - :uint64
+    - :reverseThroughputFromDestinationNode
+  151:
+    - :uint32
+    - :flowEndSecondsFromSourceNode
+  152:
+    - :uint32
+    - :flowEndSecondsFromDestinationNode
```

Comment:
I didn't remember us having all these `*SourceNode` and `*DestinationNodes` IE variations for octet and packet counts. I wonder if this is really necessary. For octant and packet counts, do we actually use all these field variations?

Reply:
Yes, we have all these [IE variations](https://github.com/antrea-io/antrea/blob/86c414e3a566538ffd8d6ca01f0e84ca25031498/build/yamls/elk-flow-collector/logstash/ipfix.yml#L114-L161). Currently the only use case is we are using  `octetDeltaCountFromSourceNode/DestinationNode` to compute node throughput in logstash. I agree these IE variations can be removed if we decide to add throughput information into the exported records. 

> could you clarify why we get a more accurate throughput calculation by performing it at the aggregator? It seems to me that the formula you give in [vmware/go-ipfix#270](https://github.com/vmware/go-ipfix/pull/270) (`diff(octetTotalCount) / diff(flowEndSeconds)`) could also be computed at the collector. 

Collector is not used in the new long-term architecture, where flow-aggregator directly exports records to a clickhouse client. 
I made it wrong in the description. Doing computation in aggregator will not improve the accuracy. The main reason for adding throughput calculation into aggregator is to avoid doing too much computation at the front-end. We want to keep most of the computation at the back-end. I'll correct the description.